pom.xml

```diff
@@ -758,7 +759,13 @@
       <dependency>
         <groupId>org.bouncycastle</groupId>
         <artifactId>bcprov-jdk15on</artifactId>
-        <version>1.65</version>
+        <version>${bouncycastle.version}</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.bouncycastle</groupId>
+        <artifactId>bcpkix-jdk15on</artifactId>
```

Comment:
just wondering: what kind of test failure came when bcpkix wasn't included?

Reply:
Here is the test failure report: https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-253/1/testReport/
Tests were complaining that they were not able to find org/bouncycastle/asn1/bsi/BSIObjectIdentifiers

Reply:
okay, thanks for clarifying!

Reply:
And with this I also checked that hadoop and other dependencies under TEZ dependency tree and they were using older version of bcpkix. So I thought to upgrade that as well